src/Shared/Resources/Strings.shared.resx

```diff
@@ -200,6 +200,10 @@
     <value>MSB5008: Error parsing the solution configuration section in solution file. The entry "{0}" is invalid.</value>
     <comment>{StrBegin="MSB5008: "}UE: The solution filename is provided separately to loggers.</comment>
   </data>
+  <data name="SolutionParseNestedProjectErrorWithNameAndGuid" Visibility="Public">
+    <value>MSB5009: Error parsing the nested project "{0}" section with GUID: "{1}" in solution file.</value>
```

Comment:
Should this also pass (and report) `ParentProjectGuid`? Maybe something like

```suggestion
    <value>MSB5009: Error parsing the project "{0}" section with GUID: "{1}". It is nested under "{2}" but that project is not found in the solution.</value>
```

?

Reply:
+1. Might as well display as much helpful information as possible.